### PR TITLE
Shopping Cart API endpoints and dummy JSON

### PR DESCRIPTION
**Pull request by Szy11mon**

### CODE REVIEW

**Comment by PiotrKolecki** [comment]
Please ask for review

File: shopping-cart-service/src/main/java/agh/fis/shopping_cart/controller/ShoppingCartController.java [hunk 1]
```diff
@@ -17,17 +21,53 @@ public ShoppingCartController( ProductCatalogClient productCatalogClient) {
...
+        item2.put("name", "GTA V");
+        item2.put("id", 4);
+        item2.put("price", 120);
+        item2.put("publisher", "Rockstar Games");
+        array.put(item2);
+
+        json.put("products", array);
+
+        message = json.toString();
+        return message;
```
Comment (DorianKossowski) [inline shopping-cart-service/src/main/java/agh/fis/shopping_cart/controller/ShoppingCartController.java:50]: Maybe try to use a data transfer object (ShoppingCartDto?) with fields like ids and a set of items or something like this.  It's not important at this stage to return some real data but to have some structures that will be visible in API. You can try with OpenAPI as I mentioned on Teams

File: shopping-cart-service/src/main/java/agh/fis/shopping_cart/controller/ShoppingCartController.java [hunk 2]
```diff
@@ -17,17 +21,53 @@ public ShoppingCartController( ProductCatalogClient productCatalogClient) {
...
+    }
+
+    @PostMapping("/")
+    public String createCart() {
         logger.info("Get on shopping cart /");
-        return "Hello ShoppingCart";
+        return String.format("Cart with id %s created", 3);
```
Comment (DorianKossowski) [inline shopping-cart-service/src/main/java/agh/fis/shopping_cart/controller/ShoppingCartController.java:56]: Try to return DTO from methods not Strings

**Review by DorianKossowski — CHANGES_REQUESTED** [review]

**Review by PawelKepka — APPROVED** [review]

File: shopping-cart-service/src/main/resources/api.yaml [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+        - date_created
+        - products
+
+    ShoppingCartCreationDto:
+      description: Keeps information needed to create cart
+      type: object
+      properties:
+        customer_id:
+          type: integer
+        products:
```
Comment (DorianKossowski) [inline shopping-cart-service/src/main/resources/api.yaml:161]: Probably cusomerId should be enough

**Review by DorianKossowski — APPROVED** [review]

**Pull request merged**